src/main/java/org/czekalski/petapiintegration/resourceServer/service/AnimalService.java

```diff
@@ -0,0 +1,7 @@
+package org.czekalski.petapiintegration.resourceServer.service;
+
+import org.czekalski.petapiintegration.resourceServer.apiResource.v1.dto.AnimalResourcesListDto;
+
+public interface AnimalService {
+    AnimalResourcesListDto findDogsByCityIdAndDogsQuantity(String cityId, String stateId, int size);
```

Comment:
Ta nazwa jest trochę myląca. Bo tak naprawdę to jest bardziej
```suggestion
    AnimalResourcesListDto findDogsByCityIdAndStateId(String cityId, String stateId, int limit);
```